src/frontend/efiling-frontend/src/components/page/upload/Upload.js

```diff
@@ -17,74 +18,192 @@ import { propTypes } from "../../../types/propTypes";
 import "./Upload.css";
 import PackageConfirmation from "../package-confirmation/PackageConfirmation";
 
-const items = [
-  "Select document description",
-  "Affidavit",
-  "Affidavit of Attempted Service",
-  "Case Conference Brief",
-];
+const filesToUpload = {
+  documents: [],
+};
+
+const checkValidityOfUploadedFiles = (setContinueBtnEnabled) => {
+  const isValid = (currentValue) =>
+    Object.prototype.hasOwnProperty.call(currentValue, "isAmendment") &&
+    Object.prototype.hasOwnProperty.call(
+      currentValue,
+      "isSupremeCourtScheduling"
+    );
+
+  if (filesToUpload.documents.length === 0) setContinueBtnEnabled(false);
+  else if (filesToUpload.documents.every(isValid)) setContinueBtnEnabled(true);
+};
```

Comment:
Nit: Should this be setting the setContinueBtnEnabled or just returning a true or false?

Reply:
awesome suggestion!